### ISSUE TITLE
Prescribing boundary conditions

### ISSUE THREAD

**Issue by Cfckoh** [weymouth/WaterLily.jl#54]
I've been playing around with the package a bit for 2D flows and know I can prescribe constant valued Dirichlet boundary conditions on the x and y boundaries using an array of length 2 (for example [1.0,0.0]) as an argument to the Simulation constructor. Is there a way to assign non constant boundary conditions and Neumann conditions? In particular I am interested in prescribing periodic conditions.

**Comment by weymouth** [comment]
Nope. We haven't implemented any domein boundary conditions yet. It's certainly past due, but it would take a bit of planning before jumping to an implementation.

**Comment by weymouth** [comment]
Some progress on periodic BCs [here](https://github.com/TzuYaoHuang/WaterLily.jl).

**Comment by weymouth** [comment]
#83 closes this
